dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.ts

```diff
@@ -64,6 +64,22 @@ const mapAmendmentToLatestCompletedFacilityEndDate = (
   };
 };
 
+const mapAmendmentToLatestCompletedBankReviewDate = (
+  amendment: TfmFacilityAmendment,
+): {
+  amendmentId: string;
+  bankReviewDate: string;
+} => {
+  const { amendmentId, bankReviewDate } = amendment;
+  if (!bankReviewDate) {
+    throw new Error('Found amendment does not have a defined bankReviewDate');
```

Comment:
Are you thinking a `.error.ts` file with an error class specifically for Facility end date / bank review date mismatch errors (same set up as e.g. `InvalidFacilityIdError`)?

Reply:
Yeah basically, it allows us to capture that information & send it in the response. I think this will currently end up as a generic 500 response & not be very helpful. What do you think?

Reply:
Discussed on call, leaving as is here as we don't expect this error to occur.